commcare_connect/opportunity/views.py

```diff
@@ -967,11 +953,62 @@ def send_message_mobile_users(request, org_slug=None, pk=None):
     )
 
 
+# used for loading api key dropdown
+@org_member_required
```

Comment:
This should probably be in the `users` app since it is tied to the user rather than the opportunity. That is a bit weird given that it is in an opportunity specific page, but still makes sense to me since the data wouldn't change with different opportunities in the URL.

Reply:
https://github.com/dimagi/commcare-connect/pull/666/commits/586d38642657d3e203e4cc48c685d3e4078151be